.github/CODEOWNERS

```diff
@@ -1,36 +1,45 @@
 # https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
 
 # Ending a path in a `/` will specify the code owners for every file
+
 # nested in that directory, on any level
 
 # Default owners
-* @pandemicsyn @edgarrmondragon
+
+- @pandemicsyn @edgarrmondragon
 
 # CI
+
 .github/actions/ @WillDaSilva @meltano/engineering
 .github/workflows/ @WillDaSilva @meltano/engineering
 
 # Discovery (End-of-Life)
+
 /src/meltano/core/bundle/discovery.yml @aaronsteers
 
 # Web App and API owners
-/src/webapp/ @alexmarple @pandemicsyn
-/src/meltano/api/ @pandemicsyn @alexmarple
```

Comment:
We probably still want secondary reviewers here. Maybe @cjohnhanson / @rwfeather since they have some FE foo as well?

/cc @aaronsteers